src/integration/resources/inprocess/dbnode_test.go

```diff
@@ -38,6 +38,13 @@ import (
 	"github.com/m3db/m3/src/x/serialize"
 )
 
+func TestNewDBNodeNoSetup(t *testing.T) {
```

Comment:
This change makes tests like this possible (i.e. creating a DB node without requiring the topology to be configured to shut it down).